### DISCUSSION TITLE
多用户多目录的权限问题

### DISCUSSION THREAD

**Discussion by myfreewiki** [AlistGo/alist#3693] [General]
1.单目录多用户的问题，管理员可以所有操作，单个用户可能只有读权限，没有写权限。目前用户只能设置全局权限。

2.非管理用户多目录的问题，目前只能挂一个目录。

对于问题1，可以多设置几个不同权限的用户(使用者估计会头大)；
对于问题2，可以给用户挂根目录，其他不想让访问的目录可以隐藏或者加密码，但要是有两个非管理员用户，同一个目录，一个可以访问，一个不能访问，就没办法设置了。

我的原始场景是这样的：家庭有多个成员，一部分挂载是只有管理员能看到能操作的，有一部分挂载是家庭成员只读的，一个公共挂载是所有人可以读写的。

欢迎大家讨论下解决方案

**Comment by ttxs25830** [comment]
这个其实很简单，第一个问题可以制作权限表或者干脆学Linux采用用户组解决，第二个问题更简单，允许像文件系统一样创建目录间的软连接即可。

**Reply by JinDasheng** [reply]
来个PR补一下啊，老板
